### ISSUE TITLE
trying to create role that already exists

### ISSUE THREAD

**Issue by tgam-smickeler** [jnonino/terraform-aws-ecs-fargate-scheduled-task#2]
I've created a task with ```jnonino/ecs-fargate-task-definition/aws``` and am now trying to use  ```jnonino/ecs-fargate-scheduled-task/aws``` to schedule a task, but it appears to be trying to create the same role that the td module has already created.

I end up with the following error:
`Error: Error creating IAM Role session-based-container-reco-ecs-task-execution-role: EntityAlreadyExists: Role with name session-based-container-reco-ecs-task-execution-role already exists.
	status code: 409, request id: b2446842-bebb-11e9-ab18-c7397f4134f6
`

code:

```
module "scheduled_task" {
  source          = "jnonino/ecs-fargate-scheduled-task/aws"
  version         = "1.0.3"
  name_preffix    = var.container_name
  profile         = var.profile
  region          = var.region
  ecs_cluster_arn = module.ecs-cluster.aws_ecs_cluster_cluster_arn
  event_rule_name = "${var.container_name}-cron"
  event_rule_schedule_expression = "cron(30 10 * * ? *)"
  event_rule_description                      = "run daily at 10:30 am"
  event_target_ecs_target_task_definition_arn = module.td.aws_ecs_task_definition_td_arn
  event_target_ecs_target_subnets             = var.private_subnets_ids
}
```

**Comment by tgam-smickeler** [comment]
If I change ```name_preffix = var.container_name``` to ```name_preffix = "${var.container_name}-scheduled"``` it works

**Comment by jnonino** [comment]
Fixed in version [1.0.4](https://registry.terraform.io/modules/jnonino/ecs-fargate-scheduled-task/aws/1.0.4)
